crates/extensions/tedge_file_system_ext/Cargo.toml

```diff
@@ -11,6 +11,7 @@ repository = { workspace = true }
 
 [dependencies]
 async-trait = "0.1"
+log = "0.4"
```

Comment:
In most of the other crates, we use `tracing` instead of `log`. This inconsistency is not just limited to this crate, but let's try and avoid it at least in newer code.

Reply:
What I heard from @didier-wenzek is that `trace` is heavy, so I am just using the `log` crate.

Reply:
I also agree that `log` is probably more that sufficient for our simple logging needs. But, the Rust ecosystem seems to be leaning more towards wider usage of `tracing` instead of `log` as you can see in https://github.com/rust-lang/compiler-team/issues/331 . Anyway, we just need to stay consistent, whichever one we choose. If the mandate is to stick to `log`, will keep that in mind.

Reply:
This is a point we need to discuss. I have a preference for `log` simply because in practice our use of `tracing` is reduced to logging without any trace.